apis/v1alpha2/policy_types.go

```diff
@@ -120,3 +122,81 @@ const (
 	// policy is attached to an invalid target resource.
 	PolicyReasonTargetNotFound PolicyConditionReason = "TargetNotFound"
 )
+
+// PolicyAncestorStatus describes the status of a route with respect to an
+// associated Ancestor.
+//
+// Ancestors refer to objects that are either the Target of a policy or above it in terms
```

Comment:
nit: `Ancestors` doesn't seem like the right phrase.  It represents parents and parents of parents ad infinitum.  A target is more akin to a predecessor.

Reply:
I've added some additional wording that I think clarifies why I've chosen this word.